halle_backup/Makefile

```diff
@@ -0,0 +1,8 @@
+DBS = $(shell cat databases.yml)
+all: $(DBS)
+$(DBS): % : dir_%
+	@$(shell make DB=$@ -f Makefile_t)
+dir_%:
+	@mkdir -p pg_dumps/$@
+clean:
+	@rm -rf ./pg_dumps
```

Comment:
Please check this tool https://github.com/lotint/ghostwriter/blob/master/tools/read_config.py
We use it to read configs from yaml file in our other project.
```make
# define work dir
WORK_DIR = /tmp/backup

# only db names here
DB_NAMES = $(shell $(CREADER) sections $(CURPATH)/databases.yaml)
# list of db targets
DBS = $(addprefix db_,$(DB_NAMES))

$(WORK_DIR):
    mkdir -p $(WORK_DIR)

db_%: | $(WORK_DIR)
    make DB=$% -f Makefile_t
```